include/bitcoin/database/databases/block_database.hpp

```diff
@@ -114,7 +114,7 @@ class BCD_API block_database
     typedef system::message::compact_block::short_id_list short_id_list;
 
     link_type associate(const system::chain::transaction::list& transactions);
-    void index(const_element& element, bool positive, bool candidate);
+    void mote(const_element& element, bool positive, bool candidate);
```

Comment:
Just a nit here.

I am not 100% sure about using method name "mote" here. I had to look it up in dictionaries in case I was unaware of mote as verb. The use of "mote" might confuse future contributors as well.

How about calling it "update", as that is what the method essentially does; or maybe even a more descriptive "update_confirmation_state"?